nixos/modules/config/fonts/fontdir.nix

```diff
@@ -29,7 +29,8 @@ in
 
       enable = mkOption {
         type = types.bool;
-        default = false;
+        default = config.services.flatpak.enable;
```

Comment:
Maybe enable it explicitly in the flatpak module. That way people will not be able to just override it accidentally. It will also be properly shown in nixos-option as defined by flatpak.